Web/starter_project_vanguard/types/NavTypes.ts

```diff
@@ -0,0 +1,4 @@
+export interface NavType {
```

Comment:
might be better to name this something else 
like NavItem or sth because navtype implies some type of navigation (you can feel free to use similar names to what is used in the UI or tailwind or sth )